src/style.css

```diff
@@ -0,0 +1,84 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+  text-align: center;
+  line-height: 300px;
```

Comment:
do not use line-height to set block height

Reply:
I use this to align the text vertically